src/functions.php

```diff
@@ -256,6 +256,13 @@ function server_supports_feature(Server $server, $feature)
     return $minWireVersion <= $feature && $maxWireVersion >= $feature;
 }
 
+/**
+ * Return whether the input is an array of strings.
+ *
+ * @internal
+ * @param array $input
+ * @return boolean
```

Comment:
Technically, `$input` is mixed although we expect it to be an array if this function is to return `true`.

If all of these functions are internal, is there any reason not to use return type hints?

Reply:
Good point. I've added native types for all internal functions (`with_transaction`) was excluded.